replica/table.cc

```diff
@@ -624,6 +624,9 @@ compaction_group& table::compaction_group_for_token(dht::token token) const noex
                                                 idx, _cg_manager->log2_compaction_groups(), _compaction_groups.size(), token));
     }
     auto& ret = *_compaction_groups[idx];
+    if (token == dht::minimum_token() || token == dht::maximum_token()) {
```

Comment:
1. Isn't maximum_token valid (and part of the last cg)?
2. The caller can be fixed too, to start at the first valid token.
3. @raphaelsc / @tgrabiec why doesn't the first compaction group start with the `minimum_token`?